apollo-router/src/snapshots/apollo_router__apollo_telemetry__test__basic_test.snap

```diff
@@ -0,0 +1,13 @@
+---
+source: apollo-router/src/apollo_telemetry.rs
+assertion_line: 279
+expression: normalized
+
+---
+
```

Comment:
This looks wrong: it should start with `# -`

Reply:
All these snapshots are "dead" placeholders waiting for apollo-rs normalisation support.

Reply:
Great; I'll be happy to review that work wherever it ends up once it starts!